Hackthebox_UniCtf_TheVault/solution.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+import r2pipe
+
+#Opens binary with r2pipe in debug mode, and analyzes it
+r = r2pipe.open('./vault')
```

Comment:
Please add this binary

Reply:
I added the binary, and the readme file.